public/people.js

```diff
@@ -0,0 +1,62 @@
+people = [
```

Comment:
Let's move this file in the `client` folder. I think webpack will have difficulty importing this if it's outside of the client folder. This is just a temporary file which will be deleted later so we can just put it directly in the client folder for now.